src/Goerli-DssSpell.sol

```diff
@@ -86,278 +27,107 @@ contract DssSpellAction is DssAction {
+            auctionDuration:       220 minutes,
+            permittedDrop:         9000,
+            liquidationRatio:      10500,
+            kprFlatReward:         300,
+            kprPctReward:          10 // 0.1%
+        });
 
-        // Rely Oracles team address in all the medians except MATIC that was created by them
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_ETH")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_BAT")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_WBTC")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_ZRX")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_KNC")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_MANA")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_USDT")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_COMP")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_LRC")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_LINK")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_BAL")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_YFI")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_UNI")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_AAVE")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
+        DssExecLib.addNewCollateral(GUNI_A);
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC1-A", 10 * MILLION, 10 * MILLION, 8 hours);
 
-        // Bump changelog version
-        DssExecLib.setChangelogVersion("1.9.5");
+        DssExecLib.setChangelogAddress("GUNIV3DAIUSDC1", GUNI);
+        DssExecLib.setChangelogAddress("MCD_JOIN_GUNIV3DAIUSDC1_A", MCD_JOIN_GUNI_A);
+        DssExecLib.setChangelogAddress("MCD_CLIP_GUNIV3DAIUSDC1_A", MCD_CLIP_GUNI_A);
+        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_GUNIV3DAIUSDC1_A", MCD_CLIP_CALC_GUNI_A);
+        DssExecLib.setChangelogAddress("PIP_GUNIV3DAIUSDC1", PIP_GUNI);
```

Comment:
We should also bump ChangeLog to `v1.9.6`

Reply:
done